common/djangoapps/enrollment/views.py

```diff
@@ -603,7 +603,7 @@ def post(self, request):
                 # We received an explicitly-linked EnterpriseCustomer for the enrollment
                 if explicit_linked_enterprise is not None:
                     try:
-                        enterprise_api_client.post_enterprise_course_enrollment(username, unicode(course_id), True)
+                        enterprise_api_client.post_enterprise_course_enrollment(username, unicode(course_id), None)
```

Comment:
I am considering removing this parameter altogether for this function. there is also the cleanup below in a todo that I could just do right now as part of this. wasn't sure how far to take this.